### PR TITLE
RPE-30: Create basic version of Express application

### PR DESCRIPTION
**Pull request by lgonczar**
This version doesn't complete the story - it's just the base for adding further changes, such as common view templates, config, quality checks, Docker setup, CI integration, localisation.

### CODE REVIEW

File: .sass-lint.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+options:
+  max-warnings: 0
+files:
+  include: 'src/**/*.scss'
```
Comment (rkondratowicz) [inline .sass-lint.yml]: is this needed? do we add any custom sass files?

Comment (lgonczar) [reply .sass-lint.yml]: We don't, but we've got sass check in place

File: src/main/app.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+expressNunjucks(app)
+
+app.use('/', RouterFinder.findAll(path.join(__dirname, 'routes')))
+
+// error handler
+app.use(function (err, req, res, next) {
+  // set locals, only providing error in development
+  res.locals.message = err.message
+  res.locals.error = env === 'development' ? err : {}
+
```
Comment (rkondratowicz) [inline src/main/app.ts]: do you plan to add logging here later?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "expressjs-template",
+  "version": "0.0.0",
```
Comment (rkondratowicz) [inline package.json]: perhaps `0.0.1`?

File: src/main/app.ts (hunk 2 above)
Comment (lgonczar) [reply src/main/app.ts]: yes

File: package.json (hunk 3 above)
Comment (lgonczar) [reply package.json]: will change

Comment (lgonczar) [reply package.json]: done

**Pull request merged**